src/src/data.ts

```diff
@@ -1,5 +1,5 @@
 import * as debug from 'debug'
```

Comment:
Do not commit those changes in this PR - they are formatting changes in file otherwise untouched by PR. Otherwise this will obscure git history.